HW5/task1.py

```diff
@@ -0,0 +1,17 @@
+# Задача 26:  Напишите программу, которая на вход принимает два числа A и B, и возводит число А в целую степень B
+# с помощью рекурсии.
+# *Пример:*
+# A = 3; B = 5 -> 243 (3⁵)
+# A = 2; B = 3 -> 8
+
+A, B = int(input('Введите число А: ')), int(input('Введите число B: '))
+
+
+def func(A, B):
+    if B == 0:
+        return 1
+    else:
+        return A * func(A, B - 1)
```

Comment:
Блок кода `else` не нужен.
```suggestion

     return A * func(A, B - 1)
```